### DISCUSSION TITLE
No more one island

### DISCUSSION THREAD

**Discussion by yodaylay22** [BG-Software-LLC/SuperiorSkyblock2#397] [Pending Feature Requests]
have a profile menu with your islands and coop islands
or 
be a member of several islands even though you have a private island of mine

**Comment by Pikaronga** [comment]
To add to what this player is saying is allow the creation of multiple profiles (like old hypixel) where one can have multiple islands and go in-between these and start over to take on the challenge from the beginning without deleting his old "main" island. This is a very good feature and would greatly add functionality to the plugin. This would also be very helpful to have apis for when switching profiles. And most likely a lot of other ones to for other plugins to hook into to make the Skyblock experience better and synced.
